pages/manage/productions/create.vue

```diff
@@ -254,63 +254,69 @@
 </script>
 
 <style lang="scss">
-  label {
-    font-size: 1rem;
-    font-weight: 700;
-  }
+  .productions-create-page {
+    .create-title {
```

Comment:
If you put this in a `scoped` style block, you wouldn't have to add the `productions-create-page` class just for this selector.

Reply:
Will be implemented in next PR